SpatialGDK/Source/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -49,6 +51,8 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject
 	Worker_ConnectionParameters CreateConnectionParameters(FConnectionConfig& Config);
 	bool ShouldConnectWithLocator();
 
+	void GetAndPrintConnectionFailureMessage();
+
 private:
```

Comment:
Drive by: If you happen to push another commit please remove this `private`. NBD if not.